packages/grid/x-grid/src/tests/apiRef.XGrid.test.tsx

```diff
@@ -0,0 +1,77 @@
+import { ApiRef, useApiRef } from '@material-ui/data-grid';
+import { XGrid } from '@material-ui/x-grid/XGrid';
```

Comment:
I would recommend disabling auto imports

```suggestion
import { XGrid, ApiRef, useApiRef } from '@material-ui/x-grid/XGrid';
```

x all the other places